src/EnergyPlus/WindowManager.cc

```diff
@@ -6942,8 +7172,29 @@ namespace WindowManager {
             }
         }
 
-        NominalConductance = 1.0 / (rOut + Rbare + rIn); // Autodesk:Uninit Rbare was uninitialized if ngllayer > 4
+        NominalConductance = 1.0 / (rOut + Rbare + rIn);
+        Real64 inputU = state.dataMaterial->Material(state.dataConstruction->Construct(ConstrNum).LayerPoint(1)).SimpleWindowUfactor;
+        // only compute adjustment ratio when there is valid user input U
+        if ((WinterSummerFlag == 1)) {
+            if (inputU > 0) {
+                state.dataHeatBal->CoeffAdjRatio(ConstrNum) = inputU / NominalConductance;
+                NominalConductance = inputU;
```

Comment:
I don't buy this at all, far too circular.  Once you get the adjustment ratio, the big block above needs to be repeated with the modification to determine the new Nominal Conductance.

Reply:
That makes more sense. Previously we assume the radiative film coefficient would also be adjusted. But seems that's not applicable in run time. I'll fix it based on the new approach.

Reply:
@EnergyArchmage I pushed the new changes. Haven't fully done with the testing yet but just wanted to make sure if my understanding of your comments here is correct. 

Previously we were trying to solve `AdjustedNominalU = InputU = NominalU * adjRatio = adjRatio * 1 / (rin + Rbare + rout)` here, assuming `adjRatio` will apply to both `hr` and `hc`, and `Rbare` is small enough to be ignored. Now with `hr` not adjustable, the current implementation solves the below equation, assuming `adjRatioOut == adjRatioIn` to make it solvable.

<img width="827" alt="Screen Shot 2021-08-20 at 10 15 24 AM" src="https://user-images.githubusercontent.com/23508095/130270202-3967b401-326b-45c2-b1c6-ad761e0aa5fb.png">

Reply:
Regarding the upper limit, since the film coefficient is evaluated as
<img width="211" alt="Screen Shot 2021-08-20 at 10 22 21 AM" src="https://user-images.githubusercontent.com/23508095/130270789-17b9c517-c8db-4420-9e55-226d176b8ba3.png">
instead of `rRad + rConv`, it's possible that the overall `r` to be evaluated to zero. I wonder how's it possible to set an psysical upper limit this way (an air void?). The limit does exist in `Rbare`. With a minimum thickness of 0.002m, there won't be an infinite total heat transfer rate.

Reply:
I don't think you can really write InputU = NominalU * adjRatio.  You need to solve for a ratio where AdjustedNominalU = InputU and AdjustedNominalU is a function of adjRatio.

Sorry I my PC is packed up.

Reply:
Yes. That's what's implemented in the latest commit. Sorry that the equation is a bit misleading. A clearer  representation should be
<img width="918" alt="Screen Shot 2021-08-20 at 11 20 21 AM" src="https://user-images.githubusercontent.com/23508095/130277111-8d89bc9d-9c44-48a2-a30c-1c1c697055d1.png">

We solve this in a while loop `while (inputU - NominalConductance > 0.01)`, with small `InputU`, we won't even get into the adjustment loop, because `NominalConductance` is evaluated to be close to `inputU` at the very first iteration.